src/Contracts/ViewModels.ts

```diff
@@ -361,6 +362,7 @@ export enum CollectionTabKind {
   Schema = 19,
```

Comment:
>Schema [](start = 2, length = 6)

interesting.. whats this?

Reply:
This is for the Parquet Schema tab which only works for accounts with analytical store enabled.